src/store/playerActions.js

```diff
@@ -116,18 +136,18 @@ export const resetCookieSuccess = () => ({
 });
 
 export const logIn = (username) => (dispatch) => {
-    axios.post('/auth/login', {
+    instance.post('/auth/login', {
         username: username
     })
-    .then(() => {
-        connectSocket();
-        dispatch(logInSuccess(username))
+    .then((res) => {
+        if (res.status === 200) {
+            connectSocket();
+            dispatch(logInSuccess(username))
+        } else {
+            dispatch(logInFailure(res.data.message));
```

Comment:
Do you need the `else` _and_ the `catch`.

Reply:
Yeah, if the error's status is different from the ones I'm validating.. for example, when it's an internal server error 500